rest-service/manager_rest/test/test_list_v1.py

```diff
@@ -57,6 +57,12 @@ def instance(*args, **kwargs):
             # restore original function
             config.instance = original_instance_func
 
+    def test_insecure_endpoints_enabled_by_default(self):
+        try:
+            self.client.events.get(execution_id='111')
+        except CloudifyClientError, e:
+            self.assertNotEquals(405, e.status_code)
```

Comment:
right.. well, i guess its ok then, but not really ideal, as theoretically there could have been an 400 or 50x error prior to the 405 validation check which would make this test pass anyway